diff --git a/node_modules/react-native-game-engine/react-native-game-engine.d.ts b/node_modules/react-native-game-engine/react-native-game-engine.d.ts
index 08d5047..ec50322 100644
--- a/node_modules/react-native-game-engine/react-native-game-engine.d.ts
+++ b/node_modules/react-native-game-engine/react-native-game-engine.d.ts
@@ -4,6 +4,7 @@
 declare module "react-native-game-engine" {
     import * as React from "react";
     import { StyleProp, ViewStyle, ScaledSize } from "react-native";
+    import { Entity } from 'react-native-game-engine';
   
     interface DefaultRendererOptions {
       state: any;
@@ -94,5 +95,15 @@ declare module "react-native-game-engine" {
     }
   
     export class GameLoop extends React.Component<GameLoopProperties> {}
+
+    export interface Entity extends EntityProperties {
+      renderer?: any;
+      update?: any;
+      [key: string]: any;
+    }
+
+    export interface Engine extends EngineProperties {
+      entities: Entity[];
+    }
   }
   
